src/api/api.cpp

```diff
@@ -1,3 +1,4 @@
+#include <migraphx/operation.hpp>
```

Comment:
Do we need this include here?

Reply:
It is auto-generated from `tools/.../api.cpp`  It is needed.